.woke.yaml

```diff
@@ -0,0 +1,9 @@
+ignore_files:
```

Comment:
Perhaps check if all of these still need to be on the ignore list? Or if we can take some off

Reply:
#49